src/test/java/net/masterthought/cucumber/TagsTest.java

```diff
@@ -72,19 +73,34 @@ public void shouldgetTotalTagScenariosFailed() {
 
     @Test
     public void shouldGetTagInfoForTag1() {
-        TagObject tagObject = reportInformation.getTags().get(0);
-        assertThat(tagObject.getTagName(), is("@tag1"));
+        List<String> tagNames = new ArrayList<>();
+        for(TagObject tagObject : reportInformation.getTags()){
+            tagNames.add(tagObject.getTagName());
+        }
+
+        assertTrue(tagNames.contains("@tag1"));
```

Comment:
you did not change test data but made validation less restrictive (from is to contains) - is that correct?